scripts/build_gh_pages

```diff
@@ -29,46 +30,37 @@ Auto-generated from commit ${BRANCH_SHA1} ("${BRANCH_SUBJECT}").
 END
 )
 
-tgz_gen() {
+doc_gen() {
   local srcdir=$1
   local docdir=$2
-  local tarfile="/tmp/oak-doc-${docdir}-${BRANCH_SHA1}.tgz"
-  # Build Cargo docs and save them off outside the repo
   (
     cd "${srcdir}" || exit
-    rm -rf target/doc
```

Comment:
Won't this now pick up any detritus in `target/doc/` for the local codebase (e.g. from a previous run of `cargo doc` without `--no-deps`, or files that used to exist but which have been deleted since)?

Reply:
I should probably have had a comment in there in the first place (rather than relying on Chesterton's Fence), but I didn't originally intend this script for public consumption&hellip;